### PR TITLE
Add Favicons To Pupil Docs Website

### PR DESCRIPTION
**Pull request by nathakits**
add favicons to pupil docs website
- add favicons
- currently manifest.json also lives in `/image/favicons` dir
   - not sure where if it should be with the favicons in `/contents` or somewhere else

### CODE REVIEW

File: images/favicons/manifest.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Pupil Labs",
+  "short_name": "Pupil Labs",
+  "display": "standalone",
+  "lang": "en-US",
+  "orientation": "portrait",
+  "theme_color": "#38ea92",
+  "theme_color": "#38ea92",
+  "description": "Pupil Labs - Open source eye tracking.",
```
Comment (willpatera) [inline images/favicons/manifest.json]: Pupil Docs - user and developer documentation for Pupil eye tracking system

File: images/favicons/manifest.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Pupil Labs",
```
Comment (willpatera) [inline images/favicons/manifest.json]: Pupil Docs

File: images/favicons/manifest.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  "short_name": "Pupil Labs",
```
Comment (willpatera) [inline images/favicons/manifest.json]: Pupil Docs

File: images/favicons/browserconfig.xml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square70x70logo src="build/media/graphics/web/favicons/mstile-70x70.png"/>
```
Comment (willpatera) [inline images/favicons/browserconfig.xml]: path is wrong.

File: images/favicons/manifest.webapp [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": 2,
+  "name": "Pupil Labs",
+  "description": "Pupil Labs Website",
+  "icons": {
+    "60": "build/media/graphics/web/favicons/firefox_app_60x60.png",
```
Comment (willpatera) [inline images/favicons/manifest.webapp]: Paths are wrong here.

File: images/favicons/manifest.webapp [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    "128": "build/media/graphics/web/favicons/firefox_app_128x128.png",
+    "512": "build/media/graphics/web/favicons/firefox_app_512x512.png"
+  },
+  "developer": {
+    "name": "Pupil Labs Dev Team",
+    "url": "https://github.com/pupil-labs/pupil-labs-website"
```
Comment (willpatera) [inline images/favicons/manifest.webapp]: url is wrong

File: images/favicons/manifest.webapp [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": 2,
+  "name": "Pupil Labs",
+  "description": "Pupil Labs Website",
```
Comment (willpatera) [inline images/favicons/manifest.webapp]: Pupil Docs

File: images/favicons/manifest.webapp [hunk 8]
```diff
@@ -0,0 +1,14 @@
+{
+  "version": 2,
+  "name": "Pupil Labs",
```
Comment (willpatera) [inline images/favicons/manifest.webapp]: Pupil Docs

**Comment by willpatera** [comment]
@nathakits also need to add the manifest and favicons to the header in docuapi template/partial. Reference this theme in a pr to docuapi and we can merge when both are ready.

**Comment by nathakits** [comment]
@willpatera - Added the manifest and favicon to `/partial/header`.
This PR and this PR - https://github.com/pupil-labs/docuapi/pull/13 - is ready to merge

**Pull request merged**